src/styles/theme.scss

```diff
@@ -375,11 +389,15 @@ h5 {
   color: $secondary-colour;
 }
 
-.card--with-icon {
+.card--with-icon, .card--alternate {
   border: 2px solid $icon-background;
   border-radius: 10px;
 }
 
+.card--alternate.info > span:last-of-type{
```

Comment:
:grimacing: with the very specific selector based on structure rather than class but I see why you choose this - and missing a space.

Reply:
the span has it's on class but that is used elsewhere and I did not want to hide that element everywhere :D